src/back-end/lib/db.ts

```diff
@@ -1303,9 +1331,9 @@ export const readOneCWUProposal = tryDb<[Id, Session], CWUProposal | null>(async
     .first();
 
   if (result) {
-    result.attachments = (await connection<{ id: Id }>('cwuProposalAttachments')
+    result.attachments = (await connection<{ file: Id }>('cwuProposalAttachments')
       .where({ proposal: result.id })
-      .select('file')).map(row => row.id);
+      .select('file')).map(row => row.file);
```

Comment:
This is what I meant earlier about I/O in TypeScript not really being helpful. Because the `knex` functions return an `any`, our type-casting doesn't add any safety.

Reply:
The type annotation is really just for documentation 🤷‍♂

Reply:
Yeah it helps for inserts and updates, but doesn't do much for selects :/